tox.ini

```diff
@@ -1,16 +1,47 @@
 [tox]
-envlist = py33,py34,py35,py36,coverage,flake8,isort
+envlist = {py33,py34,py35,py36,coverage}-{cext,nocext},flake8,isort
 
 [testenv]
+setenv =
+    cext: WEBSOCKETS_CEXT_BUILD=1
+    nocext: WEBSOCKETS_CEXT_BUILD=0
 deps =
     py33: asyncio
-commands = python -m unittest
+commands =
+    python setup.py clean --all
+    python -c "import glob, os; [os.remove(f) for files in [glob.glob('./websockets/' + ext) for ext in ['*.so', '*.pyd']] for f in files]"
```

Comment:
I tried to remain pythonic (I just started learning python, there might be something better to do) in the way `rm -rf` is done to avoid multi platform issues. The only bit that depends on a specific platform is coverage with cext that won't work on Windows.